runtime/binding-kafka/src/main/zilla/protocol.idl

```diff
@@ -510,6 +513,50 @@ scope protocol
             }
         }
 
+        scope offset_commit
+        {
+            struct OffsetCommitRequest
+            {
+                string16 groupId;
+                int32 generationId;
+                string16 memberId;
+                string16 groupInstanceId = null;
+                int32 topicCount;
+            }
+
+            struct OffsetCommitTopic
+            {
+                string16 name;
+                int32 partitionCount;
+            }
+
+            struct OffsetCommitTopicPartition
+            {
+                int32 partitionIndex;
+                int64 committedOffset;
+                int32 committedLeaderEpoch;
+                string16 committedMetadata = null;
+            }
+
+            struct OffsetCommitResponse
+            {
+                int32 throttleTimeMs;
+                int32 topicCount;
+            }
+
+            struct OffsetCommitResponseTopic
+            {
+                string16 name;
+                int32 partitionCount;
+            }
+
+            struct OffsetCommitResponseTopicPartition
+            {
+                int32 partitionIndex;
+                int16 errorCode;
+            }
+        }
+
```

Comment:
The scopes `offset_fetch` and `offset_commit` don't look so good with the underscores.
Perhaps these can go in with `group` instead as that already has multiple APIs for consumer group related concepts.